dymos/utils/testing_utils.py

```diff
@@ -0,0 +1,45 @@
+import os
+import shutil
+import tempfile
+
+
+def _new_setup(self):
+    self.startdir = os.getcwd()
+    self.tempdir = tempfile.mkdtemp(prefix='testdir-')
+    os.chdir(self.tempdir)
+    if hasattr(self, 'original_setUp'):
+        self.original_setUp()
+
+
+def _new_teardown(self):
+    if hasattr(self, 'original_tearDown'):
+        self.original_setUp()
+    self.tempdir = os.getcwd()
+    os.chdir(self.startdir)
+    try:
+        shutil.rmtree(self.tempdir)
+    except OSError:
+        pass
+
+
+def use_tempdirs(cls):
+    """
```

Comment:
Does this decorator work? I notice it isn't in any tests yet.

Reply:
Seems to but needs its own test